### PR TITLE
Adding Hindi README in root branch

### PR DESCRIPTION
**Pull request by sunn-e**
## Status
**READY**

## Description
Quick doc fixes and cration of hindi readme file

## Todos
- [ ] Documentation

## Deploy Notes
Created a file with name READMEinHindittxt.md Please make changes accordingly.

### CODE REVIEW

**Comment by adeyosemanputra** [comment]
Will move readme_jp.md or doc except readme.md into /doc/

**Comment by sunn-e** [comment]
> 
> 
> Will move readme_jp.md or doc except readme.md into /doc/

I was thinking about the same.We do not need to clutter root folder with READMEs only. @adeyosemanputra We can create a folder named README_languages to have these and upcoming READMES or help guides in fegional languages.

**Comment by rejahrehim** [comment]
Create a folder for specific languages inside doc folder and move all translations to the specific folder and link that in the Root readme

Here is the list of possible locale folders 

```python
    en-US => English (United States)
    ar-SA => Arabic (Saudi Arabia)
    bg-BG => Bulgarian (Bulgaria)
    ca-ES => Catalan (Spain)
    zh-CN => Chinese (China)
    zh-HK => Chinese (Hong Kong)    
    zh-TW => Chinese (Taiwan)
    hr-HR => Croatian (Croatia)
    cs-CZ => Czech (Czech Republic)
    da-DK => Danish (Denmark)
    nl-BE => Dutch (Belgium)
    nl-NL => Dutch (Netherlands)
    en-AU => English (Australia)
    en-CA => English (Canada)
    en-GB => English (Great Britain)
    en-GI => English (Gibraltar)
    en-HK => English (Hong Kong)
    en-IE => English (Ireland)
    en-IN => English (India)
    en-JP => English (Japan)
    en-KR => English (Korea)
    en-NZ => English (New Zealand)
    en-SG => English (Singapore)
    en-ZA => English (South Africa)
    en-TW => English (Taiwan)
    et-EE => Estonian (Estonia)
    fi-FI => Finnish (Finland)
    fr-BE => French (Belgium)
    fr-CA => French (Canada)
    fr-FR => French (France)
    de-AT => German (Austria)
    de-CH => German (Switzerland)
    de-DE => German (Germany)
    el-GR => Greek (Greece)
    he-IL => Hebrew (Israel)
    hi-IN => Hindi (India)
    hu-HU => Hungarian (Hungary)
    id-ID => Indonesian (Indonesia)
    it-IT => Italian (Italy)
    ja-JP => Japanese (Japan)
    ko-KR => Korean (Korea)
    lt-LT => Lithuanian (Lithuania)
    lv-LV => Latvian (Latvia)
    ms-MY => Malay (Malay)
    nb-NO => Norwegian Bokmal (Norway)
    nn-NO => Norwegian Nynorsk (Norway)
    pl-PL => Polish (Poland)
    pt-BR => Portuguese (Brazil)
    pt-PT => Portuguese (Portugal)
    ro-RO => Romanian (Romania)
    ru-RU => Russian (Russia)
    sk-SK => Slovak (Slovakia)
    sl-SI => Slovenian (Slovenia)
    es-AR => Spanish (Argentina)
    es-CL => Spanish (Chile)
    es-CO => Spanish (Colombia)
    es-ES => Spanish (Spain)
    es-LA => Spanish (Latin America)
    es-MX => Spanish (Mexico)
    es-US => Spanish (United States)
    sv-SE => Swedish (Sweden)
    tl-PH => Tagalog (Philippines)
    th-TH => Thai (Thailand)
    tr-TR => Turkish (Turkey)
    uk-UA => Ukrainian (Ukraine)
    vi-VN => Vietnamese (Vietnam)
```

**Comment by rejahrehim** [comment]
> > Will move readme_jp.md or doc except readme.md into /doc/
> 
> I was thinking about the same.We do not need to clutter root folder with READMEs only. @adeyosemanputra We can create a folder named README_languages to have these and upcoming READMES or help guides in fegional languages.

Done

**Comment by adeyosemanputra** [comment]
Yes @rejah we can create a folder named README_languages..

**Comment by rejahrehim** [comment]
> Yes @rejah we can create a folder named README_languages..

Done that, Please check. language specific doc folders. So if anyone redirects from root folder to language-specific folder they can have the readme displayed.

File: READMEinHindittxt.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+-   योगदानकर्ताओं के लिए, कृपया नीचे अपना नाम जोड़ें।:
+-   [Ade Yoseman](https://www.owasp.org/index.php/Ade_Yoseman_Putra)
+-   [Bambang Kurniawan](https://www.owasp.org/index.php/User:Idbmb)
+-   [Felex kemboi](https://github.com/felexkemboi/)
+-   [Lojislav Bezimenov](https://github.com/lojikil/)
+-   [Rejah Rehim](https://rejahrehim.com)
+-   [Ananthu S](https://github.com/sananthu)
+
+
+समूह चैट करें
```
[the remark this answers is not in the document]
Comment (rejahrehim) [reply READMEinHindittxt.md]: Remove extra space b/n - and text

**Comment by sunn-e** [comment]
I'm closing this because I accidently deleted the patch repo.. Will make a new PR with original documentation.

**Pull request closed without merging**